lua/kubectl/actions/informer.lua

```diff
@@ -159,7 +159,11 @@ end
 local function on_exit() end
 
 function M.start(builder)
-  if not builder.data or not builder.data.metadata then
+  local metrics = {
```

Comment:
I see what you are trying to do here and while it works it's kind of hidden behaviour, how about moving the idea to the Resourcebuilder:view and changing this instead:
Before:
```
      if opts.cmd == "curl" then
        informer.start(builder)
      end

```
After:
```
      if opts.cmd == "curl" and opts.with_informer then
        informer.start(builder)
      end

```

Reply:
done.
since i don't want to add {informer = true} to every view, I just check if it's set to false explicitly